docs/book/introduction.md

```diff
@@ -4,27 +4,178 @@ description: Welcome to ZenML!
+![ZenML Overview](.gitbook/assets/intro_zenml_overview.png)
 
-👀 Have you had trouble replicating production pipelines and monitoring models in production?
+{% tabs %}
 
-[🙇](https://apps.timwhitlock.info/emoji/tables/unicode#emoji-modal) Are you dealing with lots of unstructured notebooks or scripts with lots of manual processes?
+{% tab title="ZenML for Data Scientists" %}
 
-[🚧](https://apps.timwhitlock.info/emoji/tables/unicode#emoji-modal) Do you find it challenging to transition your ML workflows from the experimentation setting to a production-ready environment?
+As a Data Scientist, ZenML gives you the freedom to focus on modeling and
+experimentation while writing code that is immediately production-ready.
 
-If you answered yes to any of the questions above, ZenML is here to help you:
+**Develop Locally:** Since ZenML code runs in any environment, you can 
+develop ML models locally with all your favorite tools. Once you are happy with 
+your results, you can switch to a production environment via a single command 
+and all your code will just work:
 
-* **ZenML** is an **open-source MLOps framework** built for data scientists, ML Engineers, and MLOps Developers.
-* It comes equipped with a simple and flexible **Python SDK** that helps you to create **portable and production-ready pipelines**.
-* It also features **cloud-** and **tool-agnostic** **interfaces/abstractions** such as orchestration systems, artifact repositories, and model deployment platforms that cover the entire ML journey.
-* With these abstractions, it remains **extensible** and **open to customization**. That is also backed by numerous **collaborative features** that allow you to work not just with your team but also with the community.
-* Moreover, this creates a separation between the different aspects of ML workflows, allowing users to focus on creating their own respective solutions in an efficient manner.
-* Overall, it reduces the complexity of your development lifecycle, all the way **from an experimentation environment to a production-grade solution**.
+```bash
+python run.py  # develop your code locally with all your favorite tools
+zenml stack set production
+python run.py  # run on production infrastructure without any code changes
+```
+
+**Pythonic SDK:** ZenML is designed to be as unintrusive as possible. Adding 
+a ZenML `@step` or `@pipeline` decorator to your Python functions is enough to 
+turn your existing code into ZenML pipelines:
+
+```python
+from zenml import pipeline, step
+
+@step
+def step_1() -> str:
+  return "world"
+
+@step
+def step_2(input_one: str, input_two: str) -> None:
+  combined_str = input_one + ' ' + input_two
+  print(combined_str)
+
+@pipeline
+def my_pipeline():
+  output_step_one = step_1()
+  step_2(input_one="hello", input_two=output_step_one)
+
+my_pipeline()
+```
+
+**Reusability:** Defining your code as ZenML steps brings standardization 
+into your organization, allowing you to easily reuse code developed by your 
```

Comment:
'brings standardization into your organization' feels a bit vague...

Reply:
Rewrote that paragraph entirely